components/ContributorCard.js

```diff
@@ -86,7 +86,10 @@ const ContributorCard = ({ intl, width, height, contributor, currency, ...props
           </H5>
         </LinkContributor>
         <StyledTag my={2} padding="5px" letterSpacing="0.05em" fontStyle="initial">
-          {formatMemberRole(intl, getMainContributorRole(contributor))}
+          {formatMemberRole(intl, getMainContributorRole(contributor)) == 'Financial Contributor' &&
+          contributor.totalAmountDonated < 1
+            ? 'Contributor'
```

Comment:
Related to the previous feedback, we don't want to do that because we want the string to be translated for non-english users.

Reply:
Included the actual function